sriovnet_helper.go

```diff
@@ -113,30 +113,3 @@ func GetVfPciDevList(pfNetdevName string) ([]string, error) {
 	}
 	return vfDirList, nil
 }
-
```

Comment:
@moshe010 this change is unrelated to ci or travis.
Lets not mix multiple things under same subject.
Let it be a separate patch.

Reply:
recheck